doc/api/querystring.markdown

```diff
@@ -43,7 +43,12 @@ Options object may contain `maxKeys` property (equal to 1000 by default), it'll
 be used to limit processed keys. Set it to 0 to remove key count limitation.
 
 Options object may contain `decodeURIComponent` property (`decodeURIComponent` by default),
-it can be used to decode `non-utf8` encoding string if necessary.
+it can be used to decode a `non-utf8` encoding string if necessary.
+
+`querystring.unescape` will try to use `decodeURIComponent` in the
```

Comment:
I would not duplicate the description of `querystring.unescape` here, otherwise we'll have to update it twice every time it changes.